packages/backend-common/src/database/DatabaseManager.test.ts

```diff
@@ -262,7 +262,10 @@ describe('DatabaseManager', () => {
       expect(baseConfig.get().client).toEqual('sqlite3');
 
       // sqlite3 uses 'filename' instead of 'database'
-      expect(overrides).toHaveProperty('connection.filename');
+      expect(overrides).toHaveProperty(
+        'connection.filename',
+        `plugin_with_different_client/${pluginId}.sqlite`,
```

Comment:
@joeporpeglia I believe this needs a path join as well or it's going to break in the windows tests that run post merge 😅 

Look good to me otherwise 👍